docs/content/preview/architecture/docdb-sharding/colocated-tables.md

```diff
@@ -169,6 +169,18 @@ To set up xCluster for colocated tables, do the following:
     CREATE TABLE <name> WITH (COLOCATION = true, COLOCATION_ID = 20000)
     ```
 
```

Comment:
Don't see colocation_id on syntax pages for either create index or create table - is that an oversight?

Reply:
It's needed, will be done as part of https://github.com/yugabyte/yugabyte-db/issues/20582